### DISCUSSION TITLE
Drag and drop files in Quasar

### DISCUSSION THREAD

**Discussion by tianhuat** [quasarframework/quasar#11306] [General - Components / Directives / etc]
Does Quasar support Drag and Drop a file to upload like the following? 
[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/x32yB.png

**Comment by MilosPaunovic** [comment]
You're gonna have to create it yourself.

Take a look at this:
[https://quasar.dev/vue-directives/mutation#drag-and-drop-example](https://quasar.dev/vue-directives/mutation#drag-and-drop-example)

[https://github.com/quasarframework/quasar/issues/5890](https://github.com/quasarframework/quasar/issues/5890)
